service/dap/server_test.go

```diff
@@ -4694,40 +4664,37 @@ func TestSetVariable(t *testing.T) {
 
 					checkStop(t, client, 1, "main.foobar", startLineno)
 
-					// Args of foobar(baz string, bar FooBar)
-					args := tester.variables(1000)
+					// Local variables
```

Comment:
I think the original comment was more useful
You can move next line to before the comment since it is used by all checks now

Reply:
Done.